dashboard/src/pages/EventVolunteers.vue

```diff
@@ -60,52 +65,29 @@
       >
       </Dialog>
     </div>
+
     <div v-if="showAddmodal">
-      <Dialog
+      <AddMemberDialog
```

Comment:
Same thoughts here for the display of all users in the drop-down.

Reply:
It is fine in my opinion. The name and username is all that is shown.